mitiq/zne/scaling/tests/test_folding.py

```diff
@@ -1848,3 +1482,255 @@ def test_folding_keeps_measurement_order_with_qiskit():
 
     folded = fold_gates_at_random(circuit, scale_factor=1.0)
     assert folded == circuit
+
+
+def test_create_weight_mask_with_fidelities():
+    """Check the correct weight mask is created."""
```

Comment:
Not that descriptive of a comment, can be inferred from test name. Add more description or remove.